doc/testPlan

```diff
@@ -62,13 +62,13 @@
 					B) If MISS
 						1) do nothing
 				v) Snooped Write Request
+					A) Do nothing (We shouldn't have this)
```

Comment:
By we shouldn't have this, do you mean that we shouldn't test it? Is there no benefit to verifying that our cache doesn't erroneously do something when reading a command n=4 (Snooped write)?

Reply:
No, this means we should do nothing, if we see write to an address we A) shouldn't have that since its writing back to DRAM and B) nothing in our system should change. So we should test it but it should be trivial